navigation.user.js

```diff
@@ -4,70 +4,219 @@
+        hide_bride_for_extra_ration:  false,
+        hide_int_training:            false,
+        hide_gym_trainer:             false,
+        hide_lounge_buy_round:        false,
+        hide_area_tutorial_image:     false,
+        style_shop_button:            true
+    };
+    let local_storage_key = get_player_storage_prefix( "taustation_extended_nav" );
+
+    if ( window.location.pathname === "/preferences" ) {
+        add_userscript_settings( {
+            key: local_storage_key,
+            label: "Extended Naviation",
+            options: [
+                {
+                    key:     "show_change_career_links",
+                    label:   "Show career-change links",
+                    default: true,
+                    type:    "checkbox",
+                },
+                {
+                    key:   "show_careers",
+                    label: "Show Careers",
+                    type:  "array_checkbox",
+                    options: [
+                        { key: "trader",             label: "Trader" },
+                        { key: "opportunist",        label: "Opportunist" },
+                        { key: "embassy_staff",      label: "Embassy Staff" },
+                        { key: "cloning_specialist", label: "Cloning Specialist" },
+                        { key: "operative",          label: "Operative" },
+                        { key: "port_technician",    label: "Port Technician" },
+                    ]
+                },
+                {
+                    key:     "show_sub_area_nav_links",
+                    label:   "Show sub-area nav links",
+                    default: true,
+                    type:    "checkbox",
+                },
+                {
+                    key:     "show_discreet_counter",
+                    label:   "Show discreet-work counter",
+                    default: true,
+                    type:    "checkbox",
+                },
+                {
+                    key:     "show_hotel_ship_icon",
+                    label:   "Show icon when safe in Hotel Room or Ship",
+                    default: true,
+                    type:    "checkbox",
+                },
+                {
+                    key:     "show_hotel_room_message",
+                    label:   "Show large message when safe in Hotel Room",
+                    default: true,
+                    type:    "checkbox",
+                },
+                {
+                    key:     "modify_chat",
+                    label:   "Modify chat: allow maximizing chat window",
+                    default: true,
+                    type:    "checkbox",
+                },
+                {
+                    key:   "hide_bond_conversion_in_bank",
+                    label: "Hide bond-conversion in Bank",
+                    type:  "checkbox",
+                },
+                {
+                    key:   "hide_bride_for_extra_ration",
```

Comment:
s/bride/bribe/

Reply:
Ha!
Why does this make me think of dowries?...